src/HotChocolate/Core/test/Types.Tests.Documentation/WithExceptionsXmlDoc.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace HotChocolate.Types.Descriptors
+{
+#pragma warning disable 1591
+    public class WithExceptionsXmlDoc
+    {
+        /// <summary>
+        /// Query and manages users.
+        ///
+        /// You need one of the following role: Owner,
+        /// Editor, use XYZ to manage permissions.
+        /// </summary>
+        /// <returns>Bar</returns>
+        /// <exception cref="Exception" code="FOO_ERROR">Foo Error</exception>
```

Comment:
We also need to that that if the code is not provided we will not include an error

Reply:
That's already implicitly covered by other tests, but I can of course add a specific one for that :)

Reply:
An explicit test is better so when something break we know what.